backend/code_review_backend/issues/management/commands/load_issues.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import logging
+import os
+import tempfile
+
+import taskcluster
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+from django.db import transaction
+
+from code_review_backend.issues.models import Issue
+from code_review_backend.issues.models import Repository
+
+logger = logging.getLogger(__name__)
+
+INDEX_PATH = "project.relman.{environment}.code-review.phabricator.diff"
+
+
+def positive_int(x):
+    """
+    Some analyzer may give line & char positions below 0 to indicate a full file
```

Comment:
Maybe we should fix this in the bot, to have consistency.

Reply:
You're right, this is cleaner. I'm filing an issue for that (it only happened on the new task from sparky)

Reply:
https://github.com/mozilla/code-review/issues/168